packages/legend-application-studio/src/stores/graphModifier/StoreRelational_GraphModifierHelper.ts

```diff
@@ -421,3 +431,78 @@ export const rootRelationalSetImp_setPropertyMappings = action(
     );
   },
 );
+
+// --------------------------------------------- Post-Processor -------------------------------------
+
+export const relationalDatabaseConnection_addNewMapperPostProcessor = action(
```

Comment:
--> `relationalDatabaseConnection_addPostProcessor`, and pass the new post processor in as a param